cmd/lvh/runner/qemu.go

```diff
@@ -23,7 +23,7 @@ func BuildQemuArgs(log *logrus.Logger, rcnf *RunConf) ([]string, error) {
 	// quick-and-dirty kvm detection
 	if !rcnf.DisableKVM {
 		if f, err := os.OpenFile("/dev/kvm", os.O_RDWR, 0755); err == nil {
-			qemuArgs = append(qemuArgs, "-enable-kvm", "-cpu", "kvm64")
+			qemuArgs = append(qemuArgs, "-enable-kvm", "-cpu", rcnf.CPUKind)
```

Comment:
Note that the newly added -CPUKind option will only work if kvm is enabled. There is a run-time detection flag but also a configuration option. Is that what we want?

Edit: since it's mentioned that it does not have any effect when KVM is disabled, I'll assume so.